syncing/src/main/kotlin/maru/syncing/ELSyncService.kt

```diff
@@ -70,8 +70,10 @@ class ELSyncService(
     val latestBeaconBlockNumber = latestBeaconBlockHeader.number
     if (latestBeaconBlockNumber == 0UL) {
       val newELSyncStatus = ELSyncStatus.SYNCED
-      currentELSyncStatus = newELSyncStatus
-      onStatusChange(newELSyncStatus)
+      if (currentELSyncStatus != newELSyncStatus) {
```

Comment:
Minor: maybe we can move it to a separate function?

Reply:
Also, I think this deduplication of updates is breaking the tests, because I think at the time of startup not all of the handlers are set